package.json

```diff
@@ -48,7 +48,7 @@
     ]
   },
   "engines": {
-    "node": "^16.14.0 || >=18.0.0"
+    "node": "^18.17.0 || ^20.5.0 || >=22.0.0"
```

Comment:
FYI, we've standardized on `^18.17.0 || >=20.5.0` for npm 10. See https://github.com/npm/cli/issues/7754.